pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -508,3 +508,104 @@ def test_series_box_timedelta(self):
         s = Series(rng)
         assert isinstance(s[1], Timedelta)
         assert isinstance(s.iat[2], Timedelta)
+
+
+class TestTimedeltaIndexVectorizedTimedelta(object):
+    def test_contains(self):
+        # Checking for any NaT-like objects
+        # GH 13603
+        td = pd.to_timedelta(range(5), unit='d') + pd.offsets.Hour(1)
+        for v in [pd.NaT, None, float('nan'), np.nan]:
+            assert not (v in td)
+
+        td = pd.to_timedelta([pd.NaT])
+        for v in [pd.NaT, None, float('nan'), np.nan]:
+            assert (v in td)
+
+    def test_nat_converters(self):
```

Comment:
we have a section on nat tests

Reply:
There are many of these spread around.  I _really_ think we should focus for now on getting non-scalar tests out of scalar and recognize that this is a more-than-one-PR task.